src/import/load-costume.js

```diff
@@ -212,8 +212,17 @@ const loadBitmap_ = function (costume, runtime, _rotationCenter) {
             return fetched;
         })
         .then(({canvas, mergeCanvas, rotationCenter}) => {
-            // createBitmapSkin does the right thing if costume.bitmapResolution or rotationCenter are undefined...
-            costume.skinId = runtime.renderer.createBitmapSkin(canvas, costume.bitmapResolution, rotationCenter);
+            // createBitmapSkin does the right thing if costume.bitmapResolution is undefined...
+            // TODO: is rotationCenter ever undefined?
```

Comment:
Yes, the rotation center is undefined if you upload your own bitmap asset, or if you make an asset by taking a photo.